.ci/scripts/generate-cucumber-reports.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+## Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+## or more contributor license agreements. Licensed under the Elastic License;
+## you may not use this file except in compliance with the Elastic License.
+
+set -euxo pipefail
+
+## original image: ystia/cucumber-html-reporter:latest
+readonly DOCKER_IMAGE="docker.elastic.co/observability-ci/cucumber-html-reporter:latest"
+
+CUCUMBER_REPORTS_PATH="${CUCUMBER_REPORTS_PATH:-""}"
+FILES="${CUCUMBER_REPORTS_PATH}/TEST*.json"
+
+ARCHITECTURE="${ARCHITECTURE:-""}"
+PLATFORM="${PLATFORM:-""}"
+SUITE="${SUITE:-""}"
+TAGS="${TAGS:-""}"
+
+main() {
+  for f in ${FILES}
+  do
+    filename="$(basename ${f})"
+    echo "parsing ${filename}"
```

Comment:
For some reason, the glob expression is not expanded on CI. I'm able to execute this:

```
$ CUCUMBER_REPORTS_PATH=e2e/_suites/fleet SUITE=fleet ARCHITECTURE=amd TAGS=standalone PLATFORM=Darwin .ci/scripts/generate-cucumber-reports.sh 
+ readonly DOCKER_IMAGE=docker.elastic.co/observability-ci/cucumber-html-reporter:latest
+ DOCKER_IMAGE=docker.elastic.co/observability-ci/cucumber-html-reporter:latest
+ CUCUMBER_REPORTS_PATH=e2e/_suites/fleet
+ FILES='e2e/_suites/fleet/TEST*.json'
+ ARCHITECTURE=amd
+ PLATFORM=Darwin
+ SUITE=fleet
+ TAGS=standalone
+ main
+ for f in ${FILES}
++ basename e2e/_suites/fleet/TEST.json
+ filename=TEST.json
+ echo 'parsing TEST.json'
parsing TEST.json
++ pwd
++ pwd
+ docker run --rm -v /Users/mdelapenya/sourcecode/src/github.com/elastic/e2e-testing/e2e/_suites/fleet:/use/src/app/in -v /Users/mdelapenya/sourcecode/src/github.com/elastic/e2e-testing/outputs:/use/src/app/out -e CHR_APP_jsonFile=in/TEST.json -e CHR_APP_output=out/TEST.json.html -e CHR_APP_metadata_arch=amd -e CHR_APP_metadata_platform=Darwin -e CHR_APP_metadata_suite=fleet -e CHR_APP_metadata_tags=standalone docker.elastic.co/observability-ci/cucumber-html-reporter:latest

> cucumber-html-reporter@0.0.1 start /use/src/app
> node main.js

Cucumber HTML report out/TEST.json.html generated successfully.
+ for f in ${FILES}
++ basename e2e/_suites/fleet/TEST2.json
+ filename=TEST2.json
+ echo 'parsing TEST2.json'
parsing TEST2.json
++ pwd
++ pwd
+ docker run --rm -v /Users/mdelapenya/sourcecode/src/github.com/elastic/e2e-testing/e2e/_suites/fleet:/use/src/app/in -v /Users/mdelapenya/sourcecode/src/github.com/elastic/e2e-testing/outputs:/use/src/app/out -e CHR_APP_jsonFile=in/TEST2.json -e CHR_APP_output=out/TEST2.json.html -e CHR_APP_metadata_arch=amd -e CHR_APP_metadata_platform=Darwin -e CHR_APP_metadata_suite=fleet -e CHR_APP_metadata_tags=standalone docker.elastic.co/observability-ci/cucumber-html-reporter:latest

> cucumber-html-reporter@0.0.1 start /use/src/app
> node main.js

Cucumber HTML report out/TEST2.json.html generated successfully.
```

where there are a few TEST*.json files under the `CUCUMBER_REPORTS_PATH`.

But on CI, I always get:

```
[2022-03-18T09:34:29.582Z] + for f in ${FILES}
[2022-03-18T09:34:29.582Z] ++ basename 'src/github.com/elastic/e2e-testing/outputs/18.224.246.92/TEST*.json'
[2022-03-18T09:34:29.582Z] + filename='TEST*.json'
[2022-03-18T09:34:29.582Z] + echo 'parsing TEST*.json'
[2022-03-18T09:34:29.582Z] parsing TEST*.json
```

Resulting in this error:
```
[2022-03-18T09:34:36.076Z] Unable to parse cucumberjs output into json: 'in/TEST*.json' { Error: in/TEST*.json: ENOENT: no such file or directory, open 'in/TEST*.json'
```

@v1v @kuisathaverat any guess?

Reply:
I wonder if `double` quotes are needed in this case